alpha/docs.html

```diff
@@ -759,6 +759,210 @@ <h2 class="badge__basic-content-m h2">TEXT</h2>
           </div>
 
           <!-- ######################### -->
+
+          <!-- ##### COMPONENT BLOCK : BUTTONS #### -->
+
+          <div class="component--block component--typography flex-center">
+            <a class="component--title h2"># Buttons</a>
+
+            <!-- ##### COMPONENT BLOCK : BUTTONS BASIC TYPES #### -->
+
+            <div class="alert__demo-block m-1">
+              <h3 class="button__demo-title h3"># Button Types</h3>
+              <div class="button__demo-buttons flex-center">
+                <a
+                  href="#"
+                  class="uppercase h4 btn--medium border--rad-s btn__basic__alpha-text"
```

Comment:
`btn--medium border--rad-s` this class name is way too long and also not very clear what `--rad-s` means. So maybe try shorter and self-descriptive class names

Reply:
That class means border radius small . I thought of making it small but did got any good name . Should I make it just "rounded-s" 😅

Reply:
> That class means border radius small . I thought of making it small but did got any good name . Should I make it just "rounded-s" 😅

What about round-sm or rounded-sm? You can pick any depending on your choice

Reply:
ok I will do so 👍